### DISCUSSION TITLE
Spell check (english): Default treat word as stuck words

### DISCUSSION THREAD

**Discussion by SuperDre** [SubtitleEdit/subtitleedit#5623] [Ideas]
Especially with tv-shows there seem to be whole subtitles which have their words stuck in the middle of a sentence (where one probably would expect a linebreak).
I normally use Spell check to find the words and fix it, but a lot of times the suggestion selects a completely different word instead of the 'broken up' version.
So I would like it to have a preference to first select the suggestion where all the characters are the same as the word it tries to replace, but broken up. for instance 'tobe' default 'to be'.
Ofcourse a complete option (like 'Remove text for hearing impeared') to fix stuck words would be even better, but I would already be very happy if Spell check would have the split version as a suggestion (a lot of times it does it already perfectly).
Also there doesn't seem to be a suggestion for 'bea' as 'be a'..

**Comment by SuperDre** [comment]
[Battle.of.the.Bulbs.2010.1080p.WEBRip.x265-RARBG.eng.zip](https://github.com/SubtitleEdit/subtitleedit/files/7739939/Battle.of.the.Bulbs.2010.1080p.WEBRip.x265-RARBG.eng.zip)
Forgot to add an example subtitle

**Reply by niksedk** [reply]
Which lines in the attached subtitle have this issue?

**Comment by niksedk** [comment]
Good timing - just added this in the OCR fix: https://github.com/SubtitleEdit/subtitleedit/discussions/5616

Tried to add it to spell check suggestions as well:

![image](https://user-images.githubusercontent.com/1261331/147284405-04ed5b6f-ae81-45cd-98d1-691ec0e51194.png)


![image](https://user-images.githubusercontent.com/1261331/147284392-16b2858f-4d15-4f95-9345-f55988bc72e0.png)

Beta link:
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.4/SubtitleEditBeta.zip

**Comment by SuperDre** [comment]
haha, sorry, I added the subtitles I already fixed..
But tested the beta with the new Fix common errors with the OCR fix and it does clean up A LOT of the lines.
That will save a lot of time, but it still hasn't split all the words.
I've now added the original subtitles. After the OCR fix, use 'Spell check' and after a couple of skips you'll see some words still stuck.
[Battle.of.the.Bulbs.2010.1080p.WEBRip.x265-RARBG.stuck.zip](https://github.com/SubtitleEdit/subtitleedit/files/7775012/Battle.of.the.Bulbs.2010.1080p.WEBRip.x265-RARBG.stuck.zip)

But as I said, the beta already saves you A LOT of work.

**Comment by niksedk** [comment]
Thx for the example file :)
I've added some more words to the "word split list", so more lines will be split correctly.
This should be a little better: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.4/SubtitleEditBeta.zip

**Reply by SuperDre** [reply]
Oh, it's all done using a special 'word split list'? I thought it would be something like using the regular dictionary to split up words it can't recognise, as in my example the reason words are stuck (I think) is because the original linebreak was missing, so it could be any word.
